Bottom: aa184c382353db018a17a27ba119521b89a3f6f5
Top:    4340bf1e842514dbbc7f7e3d2d67d5e812f4a035
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 11:03:06 +0800

Refresh of debug.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index 97d71c6..9112918 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -1299,7 +1299,7 @@ int parse_mount(struct autofs_point *ap, const char *name,
 	 */
 	if (strstr(pmapent, "-hosts")) {
 		char *t = strstr(pmapent, "-hosts");
-		*t++ = 'h'; *t++ = 'o'; *t++ = 's'; *t = 't'; *t = 's';
+		*t++ = 'h'; *t++ = 'o'; *t++ = 's'; *t++ = 't'; *t++ = 's';
 		*t = ':';
 	}
 	p = skipspace(pmapent);
